tests/test_rabbitmq.py

```diff
@@ -0,0 +1,88 @@
+"""
+Module with unit tests for the RabbitMQ healthchecker.
+
+"""
+
+import unittest
+from unittest import mock
+
+from amqp.exceptions import AccessRefused
+
+from health_check.contrib.rabbitmq.backends import RabbitMQHealthCheck
+
+
+class TestRabbitMQHealthCheck(unittest.TestCase):
```

Comment:
Again, we don't use `UnitTest` but `pytest`. Please just use a call and switch the assert to simple `assert` instead of `self.assertEqual` and co